Sources/PackageLoading/PackageBuilder.swift

```diff
@@ -208,10 +216,12 @@ public struct PackageBuilder {
     }
 
     /// Returns path to all the items in a directory.
+    /// FIXME: This is generic functionality, and should move to FileSystem.
```

Comment:
But does a rerooted file system use relative paths?  Would it not still use regular (absolute) paths, it's just that they're all "shifted" when it actually hits the file system?  Otherwise all the APIs change, and that seems bad.  Unworkable, in fact, if we want a FileSystem protocol that can apply to a variety of file systems, rerootable or not.

I haven't followed the FileSystem discussion as much as I should; Daniel and I talked about it early on but I haven't checked recently.

The discussion in #521 seems orthogonal.  Paths are still used, even with a rerooted file system.